docs/plans/plan-finish-line.md

```
@@ -0,0 +1,178 @@
+| #32 | Inference contract tests | `work/inference-tests` | Merge after #28 |
+| #33 | Finish-line docs plan | `docs/finish-line-plan` | Merge after operational work is queued/underway |
+
+---
+
+## 3) Execution sequence (beginning to end)
+
+### Phase A — Foundation merge
+
+1. Rebase PR #29 (`work/test-phase1`) on latest `main` if needed.
+2. Run `shadow-agent` test/build gate on branch:
+   ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)